firebase-firestore/src/test/java/com/google/firebase/firestore/local/SQLiteIndexManagerTest.java

```diff
@@ -265,6 +267,19 @@ public void testCollectionGroup() {
     verifyResults(query, "coll1/doc1", "coll2/doc2/coll1/doc1");
   }
 
+  @Test
+  public void testUpdateTime() {
+    indexManager.addFieldIndex(
+        new FieldIndex("coll1")
+            .withAddedField(field("value"), FieldIndex.Segment.Kind.ORDERED)
+            .withUpdateTime(new Timestamp(10, 20)));
+
+    List<FieldIndex> indexes = ((SQLiteIndexManager) indexManager).getFieldIndexes();
+    assertEquals(indexes.size(), 1);
+    FieldIndex index = indexes.get(0);
+    assertEquals(index.getUpdateTime(), new Timestamp(10, 20));
+  }
```

Comment:
This test seems better suited for our serialization tests.

Reply:
The test here checks that the persistence queries in `addFieldIndex` and `getFieldIndex` are correct.